arangod/Cluster/FollowerInfo.cpp

```diff
@@ -279,7 +283,6 @@ bool FollowerInfo::remove(ServerID const& sid) {
   } while (TRI_microtime() < startTime + 30 &&
            application_features::ApplicationServer::isRetryOK());
   if (!success) {
-    _followers = _oldFollowers;
```

Comment:
What about checking Plan before updating Current?

Reply:
Does not really matter, we hold the lock, and do the check before communications and roll back if the plan is bad.